content/en/opentelemetry/instrument/api_support/dotnet/metrics.md

```diff
@@ -0,0 +1,92 @@
+---
+title: .NET OpenTelemetry Metrics API Support
+code_lang: metrics
+type: multi-code-lang
+code_lang_weight: 1
+further_reading:
+    - link: opentelemetry/correlate/metrics_and_traces
+      tag: Documentation
+      text: Correlate OpenTelemetry Traces and Metrics
+---
+
+## Overview
+
+{{% otel-overview-native lang=".NET" signal="Metrics" sdk_name="dd-trace-dotnet" %}}
+
+## Prerequisites
+
+- **Datadog SDK**: dd-trace-dotnet version [x.y.z] or later.
+- **OpenTelemetry API**: `OpenTelemetry.Api` version [x.y.z] or later. (The Datadog SDK provides the implementation for this API).
+- **An OTLP-compatible destination**: You must have a destination ready to receive OTLP data, such as the Datadog Agent or OpenTelemetry Collector.
+
+## Setup
+
+Follow these steps to enable OTel Metrics API support in your .NET application.
+
+1. Install the Datadog SDK:
+   ```sh
+   [TODO: Code example]
```

Comment:
Installing the .NET Tracer is not a one-line instruction like other languages. Can we link to https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-framework/?tab=nuget#install-the-tracer for .NET Framework and https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-core?tab=windows#install-the-tracer for .NET Core?